markers/polarion.py

```diff
@@ -13,3 +15,13 @@ def extract_polarion_ids(item):
         return None
 
     return map(str, polarion.args)
+
+
+@pytest.fixture(autouse=True)
+def _update_polarion_in_junit(request, record_xml_property):
+    """Adds the supplied test case id to the xunit file as a property"""
+
+    record_id_to_xml = partial(record_xml_property, "test_id")
+    ids = extract_polarion_ids(request.node)
+    if ids is not None:
+        map(record_id_to_xml, ids)
```

Comment:
**required** turn this into a real loop, map only works eagerly on python2, this would turn into a noop due to iterative map on python3, and we plan to move to python3

Reply:
working on this...

Reply:
Done